### PR TITLE
Eb 78 change logo

### PR DESCRIPTION
**Pull request by Hesoyam01**

### CODE REVIEW

File: src/app/features/kanban/components/upload-image/upload-image.component.html
Comment (J-C-V) [inline src/app/features/kanban/components/upload-image/upload-image.component.html:1]: Pls give your component some sort of form - Right now it's just a row:

image

Maybe something like this, but feel free to adjust it yourself:

image

- Upload button isn't recognizable as button (Give it at least primary color)
- We use English as ui language

File: src/app/features/kanban/components/upload-image/upload-image.component.ts
Comment (J-C-V) [inline src/app/features/kanban/components/upload-image/upload-image.component.ts:1]: - Please add comments on what your methods are doing (Is needed for submitting the documentation...)
- Specify for the user which file types are supported (line 78)

**Review by J-C-V — CHANGES_REQUESTED** [review]
Please implement a component (board-logo) which just loads and displays the board image as small logo with a given board id, so that we can easily display a board logo anywhere in the app.

File: src/app/shared/components/nav-bar/nav-bar.component.html
[the remark this answers is not in the document]
Comment (Hesoyam01) [reply src/app/shared/components/nav-bar/nav-bar.component.html:1]: done

File: src/app/features/kanban/components/upload-image/upload-image.component.ts
Comment (Hesoyam01) [reply src/app/features/kanban/components/upload-image/upload-image.component.ts:1]: @J-C-V The select file dialog only lets you select the specified type

Comment (Hesoyam01) [reply src/app/features/kanban/components/upload-image/upload-image.component.ts:1]: done

File: src/app/features/kanban/components/upload-image/upload-image.component.html
Comment (Hesoyam01) [reply src/app/features/kanban/components/upload-image/upload-image.component.html:1]: language browser specific.

Comment (Hesoyam01) [reply src/app/features/kanban/components/upload-image/upload-image.component.html:1]: done

**Review by J-C-V — APPROVED** [review]
@Hesoyam01 Current branch gets merged. Please make a new pull request for the missing board-logo component.

**Pull request merged**